src/Hl7.Fhir.Base/FhirPath/Expressions/ExpressionNode.cs

```diff
@@ -305,6 +305,11 @@ public VariableRefExpression(string name, ISourcePositionInfo location = null) :
         {
             Name = name ?? throw Error.ArgumentNull("name");
         }
+        
+        internal IEnumerable<ITypedElement> ResolveVariable(Closure context, IEnumerable<Invokee> _)
+        {
+            return context.ResolveValue(Name) ?? throw Error.InvalidOperation($"Variable {Name} not found in environment");
```

Comment:
This should return null now, otherwise the chaining to the resolve in the symbol table will never be called....  or, to avoid nulls, make it TryResolveVariable....

Reply:
It seems like this was dead code. The function call was pretty useless anyways, so I guess we unintentionally got rid of it, but it is cleaner without this method anyways :)